inventory/kubejetstream/cluster.tfvars

```diff
@@ -41,6 +41,50 @@ number_of_k8s_nodes_no_floating_ip = 0
 
 flavor_k8s_node = "4"
 
+# # Uncomment when all nodes will be GPU nodes
+# # If you wish to use this var for another reason, add the ansible groups as a comma seperated list
+# # E.g "additional-group-1,additional-group2,etc"
+# supplementary_node_groups = "gpu-node"
+
+# BEGIN HYBRID CLUSTER CONFIG
+
+# # Set to true by default, but we make it explicit here
+# port_security_enabled = true
+
+# # Must be uncommented and set to 0 to use the k8s_nodes variable
+# number_of_k8s_nodes = 0
+# number_of_k8s_nodes_no_floating_ip = 0
+
+# # "<cluster-name>-k8s-node-" will be prepended to each key name and used to create the instance name.
+# # E.g the first item below would result in an instanced named "<cluster-name>-k8s-node-nf-cpu-1"
```

Comment:
I think in the default configuration we do not have `-nf-` but I can modify this myself

Reply:
I believe the `-nf-` is added by Terraform to any node without a floating IP. In the example I was following that convention. It looks like [this line](https://github.com/zonca/jetstream_kubespray/blob/8cda1ef74d30d22058737e9dca250aae4296447e/contrib/terraform/openstack/modules/compute/main.tf#L547) is where this gets done.